lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -321,17 +359,45 @@ def headers
         }
       end
 
-      def url(_post, action, authorization)
-        if %i[authorize purchase credit].include?(action)
+      def public_key_headers
+        {
+          'Authorization' => @options[:public_key],
+          'Content-Type' => 'application/json;charset=UTF-8'
+        }
+      end
+
+      def tokenize(payment_method, options = {})
+        post = {}
+        add_authorization_type(post, options)
+        add_payment_method(post, payment_method, options)
+        add_customer_data(post, options)
+        begin
+          r = parse(ssl_post(url(:tokens, nil), post[:source].to_json, public_key_headers))
+          succeeded = success_from(:tokens, r)
+        rescue ResponseError => e
+          raise unless (400..499).cover?(e.response.code.to_i)
+
+          r = parse(e.response.body, error: e.response)
+        end
```

Comment:
I think we should be able to just call `commit` and use the existing request logic there. And where your conditional for `unstore` handles a slimmed down response, we can likewise, do the same for this transaction type.